server/authz/errors.go

```diff
@@ -34,7 +34,7 @@ func ForbiddenWithInternal(internal string, subject *fleet.User, object, action
 }
 
 // Error implements the error interface.
-func (e *Forbidden) Error() string {
+func (e Forbidden) Error() string {
```

Comment:
reason: so it actually implements the `Error` interface and can be properly detected in `transport.go`

Reply:
Am not following why your PR needs this change. (Am worried something else might break?)

Reply:
you know what, I think you're right and I was just confused. Reverted